ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -191,17 +192,16 @@ def logaddexp2(
 
 
 def diff(
-    x: Union[paddle.Tensor, list, tuple],
+     x: Union[paddle.Tensor, list, tuple]=paddle.to_tensor(),
     /,
     *,
     n: int = 1,
     axis: int = -1,
-    prepend: Optional[Union[paddle.Tensor, int, float, list, tuple]] = None,
-    append: Optional[Union[paddle.Tensor, int, float, list, tuple]] = None,
```

Comment:
here we need to convert prepend to tensor because if we pass list in paddle.gcd([1,2],[1,4). it raises error.

Reply:
discard any changes in the function declaration